Python/ceval.c

```diff
@@ -9,6 +9,7 @@
 #include "pycore_cell.h"          // PyCell_GetRef()
 #include "pycore_ceval.h"
 #include "pycore_code.h"
+#include "pycore_compile.h"
```

Comment:
Maybe add a comment why this is needed? Presumably something in generated_cases.c.h?

Reply:
It's not needed any more, I should have removed it. Thanks for the review!